dushimeemma/fs-simba-cal
# ES-3 Implement user interface

Description
This PR introduce all the necessary UI
[ ] Bug fix (Non breaking change that fixes an existing issue)

[x] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested?
[ ] Unit tests

[ ] Component Tests

[ ] Integration Tests

How Should This Be Manually tested:

- Clone this repo `git clone https://github.com/dushimeemma/fs-simba-cal/`.

- Checkout the branch `git checkout feature/ESL-3-ui`

- Run `yarn install`

- Run `yarn dev`

Checklist:
[x] My code follows all the styles and conventions outlined for this project

[x] I have linted my code before submission

[x] I have made my corresponding changes to the documentation

[x] My changes generates no new warning

[x] I have added tests that prove my feature is working or my fix is effective

[x] New and existing tests pass locally with my changes

JIRA
[ESL-3](https://www.notion.so/UI-pages-components-8eb7aebdb5da4909aa9db04291a965fb)
